src/main/java/com/cloudbees/plugins/deployer/engines/Engine.java

```diff
@@ -232,76 +261,26 @@ public void log(String message) {
         throw new DeployException("Deployment hosts of type " + configuration.getClass() + " are unsupported");
     }
 
-    public static class FingerprintDecorator implements Callable<DeployedApplicationLocation, IOException> {
```

Comment:
Also incompatible

Reply:
Allows us to delete [this nonsense](https://github.com/jenkinsci/jenkins/blob/8189c2cd9a2cb0a4e6d2dcf341fb818dbd9165ba/core/src/main/resources/jenkins/security/s2m/callable.conf#L13-L14).